dice_roll.py

```diff
@@ -2,14 +2,20 @@
 
 
 def roll_Dice():
-    print('Press R to roll the dice.\nPress Q if you want to quit this game.')
+    print('Press 1 to roll the dice.\nPress 2 if you want to quit this game.')
     while True:
-        roll = input().upper()
-        if roll == 'R':
-            print('You got number', random.randint(1, 6))
-        elif roll == 'Q':
-            print('Game Ended')
-            break
+        try:
```

Comment:
No need of enclosing the while logic inside the try expect block. we could have simply used:

```
try:
    roll = int(input())
except ValueError:
    print("Invalid input! Try Again")
    continue
if roll == 1:
    ....
```

Reply:
Also note that instead of `except exception`, I have used `except ValueError` it is a better practice to handle the specific error instead of catching the whole Exception.